<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>639</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>639</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RGB Picker Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="sliderRedValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderGreenValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBlueValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>27</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinRedValue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinGreenValue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBlueValue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QFrame" name="frameColorPreview">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="textHtmlColor">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderRedValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinRedValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGreenValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinGreenValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlueValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBlueValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
